buildbot-host/buildmaster/master.cfg

```diff
@@ -659,6 +659,36 @@ factories.update(
 )
 del factory
 
+# OpenVPN 2 clang + asan test
+clang_env = { "CFLAGS": "-fsanitize=address,undefined -fno-sanitize-recover=all -fno-omit-frame-pointer -O2", "CC": "clang" }
+clang_env.update(ccache)
+clang_env.update(lwip_env)
+
+factory = util.BuildFactory()
+factory = openvpnAddCommonUnixStepsToBuildFactory(factory, "", clang_env)
+factory = openvpnAddUnixCompileStepsToBuildFactory(factory, "", clang_env)
+factory = openvpnAddDeviceSetupStepsToBuildFactory(factory, "", clang_env)
+
+if openvpn_run_tserver_null_tests: factory = openvpnAddTServerNullPreStepsToBuildFactory(factory, "", clang_env)
+if openvpn_run_tclient_tests:      factory = openvpnAddTClientPreStepsToBuildFactory(factory, "", clang_env)
+
+factory = openvpnAddCheckStepsToBuildFactory(factory, "", clang_env)
+
+if openvpn_run_tclient_tests:      factory = openvpnAddTClientPostStepsToBuildFactory(factory, "", clang_env)
+
+factory_name = "clang"
+factories.update(
+    {
+        factory_name: {
+            "factory": factory,
+            "os": "unix",
+            "types": ["clang"],
```

Comment:
should be `["openvpn", "clang"]` for consistency with other factories.

Reply:
Fixed.